docs/Tutorials and concepts/Concepts/Core/Mask.md

```diff
@@ -0,0 +1,44 @@
+n image processing, a mask is a binary image or a matrix, where each pixel value indicates whether the corresponding pixel in the target image should be retained (usually denoted as 1 or "on") or ignored (usually denoted as 0 or "off")
+
+Creation of the mask is based on the threshold that define the level of grey intensity that separates the white pixels from the black pixels.
+
+Mask is an object which represents a binary image, therefore its properties are the same as in `Image` class. However, the methods can be slightly different.
+
+:::info
+To look deeper into Mask's properties and methods take a look at [the Mask description](../../../API%20reference/Mask/) in API reference.
+:::
```

Comment:
There seems to be a problem with this link:

![CleanShot 2023-09-22 at 12 44 19](https://github.com/image-js/image-js-docs/assets/4118690/915381c9-446f-4bb5-bb9e-d30fb586675c)